dev-docs/bidders/lasso.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: Lasso
+description: Prebid Lasso Bidder Adaptor
+pbjs: true
+pbs: false
+media_types: banner
+biddercode: lasso
```

Comment:
We recommend to add as much information as possible here to allow publishers and sales houses make better decisions.

```
gdpr_supported: true/false
gvl_id: none
usp_supported: true/false
coppa_supported: true/false
safeframes_ok: true/false
deals_supported: true/false
floors_supported: true/false
fpd_supported: true/false
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
```

Those are very relevant information

Reply:
I added more info, should be good now. Thank you for advice.